config/index.js

```diff
@@ -291,6 +291,12 @@ var conf = convict({
     format: Boolean,
     env: 'LOCKOUT_ENABLED',
     default: false
+  },
+  openIdProviders: {
+    doc: 'root urls of allowed OpenID providers',
+    format: Array,
+    default: [],
+    env: 'OPENID_PROVIDERS'
```

Comment:
I feel like the nested config namespaces were a mistake in the first place, prettier but not useful